server/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/TermsAggregatorFactory.java

```diff
@@ -47,6 +49,8 @@
 public class TermsAggregatorFactory extends ValuesSourceAggregatorFactory {
     static Boolean REMAP_GLOBAL_ORDS, COLLECT_SEGMENT_ORDS;
 
+    protected static final Logger logger = LogManager.getLogger(TermsAggregatorFactory.class);
```

Comment:
I'd go with `private` just for a bit of paranoia.

For what it's worth `LogManger.getLogger()` is very very fast. I still use the `getLogger(class)` mostly though.